helm/private/runner/runner_wrapper.sh.template

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export HELM_BIN="{HELM_BIN}"
```

Comment:
if all you need are environment variables why not use `RunEnvironmentInfo` and write args to a text file? You could even take advantage of `ctx.actions.args`

Reply:
I didn't know about that provider, good to know.